yield_plugin/src/yield_plugin.cpp

```diff
@@ -100,26 +100,49 @@ namespace yield_plugin
   {
     cav_msgs::TrajectoryPlan trajectory_plan;
     std::vector<lanelet::BasicPoint2d> map_points;
-    lanelet::BasicPoint2d first_point;
 
-    first_point.x() = ecef_trajectory.location.ecef_x * tf.transform.translation.x;
-    first_point.y() = ecef_trajectory.location.ecef_y * tf.transform.translation.y;
+    tf2::Stamped<tf2::Transform> transform;
+    tf2::fromMsg(tf, transform);
```

Comment:
Does this include the changes from #1226 ? These look very similar to that branch. If this includes both changesets let's just merge this one after addressing the review comments from that other PR. If we're mixing and matching like that though, let's also note in the PR body which other branches/PRs it relates to and constraints on merge order/conflict resolution/etc.